stag-library-compiler/src/test/java/com/vimeo/stag/processor/ElementUtilsUnitTest.kt

```diff
@@ -48,6 +47,16 @@ class ElementUtilsUnitTest : BaseUnitTest() {
 
     @Test
     fun testGetTypeFromQualifiedName() {
+        fun <T> testTypeMirrorCreationAndEquality(clazz: Class<T>) {
```

Comment:
1. Only tests that doesn't use a "nice" name (method name with spaces).
2. Might actually be worth breaking this up into 2 tests (`fun test creation equality` and `fun test creation equality`), since testing equality and inequality are 2 different things.
3. Optional; might be easier to mentally parse something like:
```kotlin
listOf(
    String::class.java,
    Any::class.java,
    ArrayList::class.java,
    DummyConcreteClass::class.java,
    DummyGenericClass::class.java,
    DummyInheritedClass::class.java
).forEach { clazz ->
    assertEquals(Utils.getTypeMirrorFromClass(clazz), ElementUtils.getTypeFromQualifiedName(clazz.name))
}
```
(not sure what the nested function offers over this).